.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,48 @@
+---
+name: bug
+title: Report a bug in Ibex
+about: Have you found a bug in Ibex or associated tooling?
+labels: Type:Bug
+---
+
+<!--
+Thank you for your interest in Ibex. Are are continuously improving Ibex, and your bug report will help us with that. To help us understand and reproduce your problem as quickly as possible we have prepared a little template below.
+
+All text in blocks starting with < !-- won't be shown in the final bug report.
+
+You can also copy/paste images into the bug report, and you can attach files to it by simply dragging them into the issue window.
+-->
+
+## Observed Behavior
+<!-- Please describe the problem you saw. Include waveforms and log files if possible. -->
```

Comment:
waveforms - ideally an .fst or gzipped .vcd (you can attach to the github issue with drag/drop)

Reply:
though waveform images are also useful and easier to deal with initially, the issue is they often leave off pertinent signals. Clearly we need to spend lots of time building some kind of .vcd/.fst renderer that integrates nicely with github issues.